Update readme for npm install error due to incorrect Node version

Adding this issue for future reference. I went through the initial setup with Node v19 and got the below error. Going back through I see the dependencies specifically list Node v14, but I think adding a note in the troubleshooting section could help others quickly fix if they come across this issue.

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: planet@0.16.69
npm ERR! Found: @angular/compiler@10.2.5
npm ERR! node_modules/@angular/compiler
npm ERR!   @angular/compiler@"10.2.5" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/compiler@">=2.3.1 <10.0.0 || >9.0.0-beta <10.0.0 || >9.1.0-beta <10.0.0 || >9.2.0-beta <10.0.0" from codelyzer@5.2.2
npm ERR! node_modules/codelyzer
npm ERR!   dev codelyzer@"^5.1.2" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! /home/pbert/.npm/_logs/2025-01-22T17_18_22_981Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/pbert/.npm/_logs/2025-01-22T17_18_22_981Z-debug-0.log
```

--- a/README.md
+++ b/README.md
@@ -158,4 +158,8 @@ npm i mime@3.0.0
 npm i @types/mime@3.0.0
 ```
 
+### Error on initial npm install
+
+If your npm install fails on your first try, first check if you are using Node v14. Other versions of Node may throw errors when installing dependencies.
+
 This project is tested with [BrowserStack](https://www.browserstack.com/).

